deploy/helm/flightctl/README.md

```diff
@@ -308,6 +308,12 @@ For more detailed configuration options, see the [Values](#values) section below
 | periodic.image.image | string | `"quay.io/flightctl/flightctl-periodic"` | Periodic container image |
 | periodic.image.pullPolicy | string | `""` | Image pull policy for periodic container |
 | periodic.image.tag | string | `""` | Periodic image tag |
+| telemetryGateway.additionalRouteLabels | string | `nil` |  |
+| telemetryGateway.config.export.prometheus | string | `"0.0.0.0:9464"` |  |
+| telemetryGateway.config.listen.device | string | `"0.0.0.0:4317"` |  |
```

Comment:
I don't see a reason for these to be configurable, we don't do so, either for other services.